trunk/Bricksmith/Source/LDraw/Support/LDrawGLRenderer.m

```diff
@@ -139,8 +139,13 @@ - (void) prepareOpenGL
 	glEnableClientState(GL_VERTEX_ARRAY);
 	glEnableClientState(GL_NORMAL_ARRAY);
 	glEnableClientState(GL_COLOR_ARRAY);
-	
-	[self setBackgroundColorRed:1.0 green:1.0 blue:1.0]; // white
+
+	// FIXME: This currently seems to override any previously saved custom background color chosen by the user.
```

Comment:
Odd. The user's background color is supposed to get pulled via `-[LDrawGLView takeBackgroundColorFromUserDefaults]`, which looks like it should get called after this does. Feel free to investigate!

Reply:
Yeah, I'll have to look into this and see what's going on.